include/dxc/DXIL/DxilConstants.h

```diff
@@ -60,6 +60,22 @@ inline int CompareVersions(unsigned Major1, unsigned Minor1, unsigned Major2,
   return 0;
 }
 
+// Utility for updating major,minor to max of current and new.
+inline bool MaxOfShaderModels(unsigned &major, unsigned &minor,
```

Comment:
MaxOfShaderModels( ) returns a bool, but it looks like the code below never checks that return.  Could this just be void or is there really something the code below needs to be aware of if 'false' is returned?

Reply:
I see in other areas that use this helper, they do in fact check the bool which means that values have changed.

Reply:
Yeah, the way I tried to separate different changes made it so the first one didn't need the return value, but other later uses did.